src/components/UserInfo/UserInfo.tsx

```diff
@@ -1 +1,12 @@
-export const UserInfo = () => {};
+import React from 'react';
+import { User } from '../../types/user';
+
+type Props = {
+  user: User | undefined;
+};
+
+export const UserInfo: React.FC<Props> = ({ user }) => (
+  <a className="UserInfo" href={`mailto:${user?.email}`}>
+    {user?.name}
+  </a>
+);
```

Comment:
```suggestion
type Props = {
  user: User;
};

export const UserInfo: React.FC<Props> = ({ user }) => (
  <a className="UserInfo" href={`mailto:${user.email}`}>
    {user.name}
  </a>
);
```
let's render this component only when a user exists